web/themes/joinup/prototype/sass/print.sass

```diff
@@ -0,0 +1,22 @@
+@media print
```

Comment:
This is fine, though I'd suggest you the following:

- make a mixin:
```
@mixin hidden-print() {
  @media print {
    display: none !important;
  }
}
```
- hide in a per-component basis:
```
@include ecl-hidden-print();
```
Optionally, you can make also a class like `.hidden-print` or something similar using this mixin.

Main goal: make your solution more flexible for others.